Tests/test_Mafft_tool.py

```diff
@@ -87,7 +87,7 @@ def test_Mafft_simple(self):
         # became "Progressive alignment 1/2..." and "Progressive alignment 2/2..."
         self.assertTrue(("Progressive alignment ..." in stderrdata) or
                         ("Progressive alignment 1/" in stderrdata), stderrdata)
-        self.assertTrue("$#=0" not in stderrdata)
+        self.assertNotIn("$#=0" , stderrdata)
```

Comment:
You have an extra space before the comma, which is why TravisCI's pep8 check is failing (here and elsewhere)